### ISSUE TITLE
Preferred source link format for a GloBI record?

### ISSUE THREAD

**Issue by jhammock** [jhpoelen/eol-globi-data#304]
If a dissemination venue would like to point to GloBI as a proximal source for data, (in addition to the GloBI data provider and associated references) what is the preferred destination? I think pointing to a browser result for a specific record is not supported, but fairly narrow filters could be appropriate, eg:

http://www.globalbioticinteractions.org/browse/index.html?accordingTo=Fernanda%20V.%20Costa.%202016&interactionType=eats&resultType=json&sourceTaxon=Brachymyrmex%20pictus&targetTaxon=Croton

broader filters can also be interesting and conducive to exploring, eg: http://www.globalbioticinteractions.org/browse/index.html?sourceTaxon=Brachymyrmex%20pictus

Possibly the browser should not be relied on in the medium or long term? In such cases, should http://www.globalbioticinteractions.org/about.html be considered?

Just looking for guidance...

**Comment by jhpoelen** [comment]
Individual records can be downloaded using the web api, and, as you noted, no visualization (html) currently exists for individual records. 

At some point, I hope that pages the http://www.globalbioticinteractions.org/browse and http://globalbioticinteractions.org may be merged.  I consider them to be UI prototypes with slightly different use cases. The browser is optimized for network visualization and the basic viewer is optimized for connecting source references to specific interactions.

I'd favor url links like http://globalbioticinteractions.org?sourceTaxon=Brachymyrmex%20pictus or even http://globalbioticinteractions.org?sourceTaxon=EOL:470412 for sake of url shortness. For lookup by reference or data source using the accordingTo, I prefer using dois (e.g. http://www.globalbioticinteractions.org/?interactionType=interactsWith&accordingTo=http%3A%2F%2Fdx.doi.org%2F10.5063%2FF19C6VB4) and dataset ids (e.g. http://www.globalbioticinteractions.org/?accordingTo=globi%3Ajhammock%2FLayman-and-Allgeier-Lionfish&interactionType=interactsWith) over citation strings or data source descriptions. 

We can always implement some automated redirect if needed. If more specificity is needed.

Does that help?

**Comment by jhammock** [comment]
Yes, I think sourceTaxon ID only as a broad option or sourceTaxon ID & dataset ID for a narrower option are easy and useable. My immediate ambition is georeferenced GloBI records for Fresh Data, see test dataset 
http://opendata.eol.org/dataset/globi/resource/a726c01b-70e3-4866-afeb-a09748f095a2

For the Fresh Data use case, I think the narrower option makes more sense. Any thoughts?

**Comment by jhpoelen** [comment]
Hmm. I can see interactions be treated as traits (weird kind of body weight - instead of bodyweight 20 kg, eats Lupus canis) or occurrence records (with dwca's associated taxa/ associated occurrence fields).

Need to think about this some more as the use case is not entirely clear to me.

**Comment by jhammock** [comment]
you know, maybe it's not so important for each record to have a unique url as it is that there's a consistent way of selecting the "best" (most informative?) url for the dataset. There are several options now- the github repo, the zenodo file, and the original dataset doi if available. Maybe one of those- or a pecking order- would be most appropriate as a source link? 

(Without losing the citation currently displayed as Source in the EOL export, of course. That info should always go somewhere. For EOL it could be http://purl.org/dc/terms/bibliographicCitation, which I think is currently unoccupied?)
